### ISSUE TITLE
[Enhancement] String To Numbers Conveters

### ISSUE THREAD

**Issue by Hackavist** [xamarin/XamarinCommunityToolkit#438]
## Summary

2 simple converters to convert from a string to int and to double right from the XAML without the need to a View model

## API Changes

StringToIntConverter / StringToDoubleConverter

## Intended Use Case

Convert from a string to int and to double right from the XAML without the need to a View model

## Who Will Do The Work?

- [x] I am willing to take this on myself
- [ ] Just putting this out there for someone to pick up

**Comment by AndreiMisiukevich** [comment]
Hello) thanks for your interest to this project)

Could you please provide any examples where it can be useful?

Thanks!

**Comment by Hackavist** [comment]
I have 2 scenarios in mind 
1) when you are displaying some info or a number based chart based on a user numerical input.
2) when u are trying to use another converter or control that only accept numbers (view to view binding)
``` XAML
 <StackLayout
            Padding="10,10"
            HorizontalOptions="Fill"
            Spacing="10"
            VerticalOptions="Fill">
            <Label Text="{rsx:Translate DoubleToIntConverterLongDescription}" TextColor="{StaticResource NormalLabelTextColor}" />
            <Label Text="{rsx:Translate TextEntryPromt}" TextColor="{StaticResource NormalLabelTextColor}" />

            <Entry
                x:Name="ExampleText"
                Placeholder="Example 2.3"
                Text="16.67"
                TextColor="{StaticResource NormalLabelTextColor}" />
            <Label
                BindingContext="{x:Reference Name=ExampleText}"
                Text="{Binding Path=Text, Converter={StaticResource DoubleToIntConverter}, ConverterParameter=2}"
                TextColor="{StaticResource NormalLabelTextColor}" />
        </StackLayout>
```
whenever you try to run this your are going to be faced with "Value is Not A valid Double"

**Comment by AndreiMisiukevich** [comment]
@Hackavist thanks! )

**Comment by Hackavist** [comment]
So can u mark it as ready to implement?

**Comment by jfversluis** [comment]
Fixed in #447

**Comment by Hackavist** [comment]
> I have 2 scenarios in mind  @AndreiMisiukevich 
> 
> 1) when you are displaying some info or a number based chart based on a user numerical input.
> 
> 2) when u are trying to use another converter or control that only accept numbers (view to view binding)
> 
> ``` XAML
> 
>  <StackLayout
> 
>             Padding="10,10"
> 
>             HorizontalOptions="Fill"
> 
>             Spacing="10"
> 
>             VerticalOptions="Fill">
> 
>             <Label Text="{rsx:Translate DoubleToIntConverterLongDescription}" TextColor="{StaticResource NormalLabelTextColor}" />
> 
>             <Label Text="{rsx:Translate TextEntryPromt}" TextColor="{StaticResource NormalLabelTextColor}" />
> 
> 
> 
>             <Entry
> 
>                 x:Name="ExampleText"
> 
>                 Placeholder="Example 2.3"
> 
>                 Text="16.67"
> 
>                 TextColor="{StaticResource NormalLabelTextColor}" />
> 
>             <Label
> 
>                 BindingContext="{x:Reference Name=ExampleText}"
> 
>                 Text="{Binding Path=Text, Converter={StaticResource DoubleToIntConverter}, ConverterParameter=2}"
> 
>                 TextColor="{StaticResource NormalLabelTextColor}" />
> 
>         </StackLayout>
> 
> ```
> 
> whenever you try to run this your are going to be faced with "Value is Not A valid Double"

@AndreiMisiukevich this the the code sample i mentioned in #494
